arangod/GeneralServer/GeneralServerFeature.cpp

```diff
@@ -386,8 +386,11 @@ void GeneralServerFeature::defineHandlers() {
                                     RestHandlerCreator<RestSimpleHandler>::createData<aql::QueryRegistry*>,
                                     queryRegistry);
 
-  _handlerFactory->addPrefixHandler(RestVocbaseBaseHandler::TASKS_PATH,
-                                    RestHandlerCreator<RestTasksHandler>::createNoData);
+  if (server().isEnabled<V8DealerFeature>()) {
```

Comment:
I would prefer something that errors out with "not supported without V8" so the user has a meaningfull error message.